### PR TITLE
Bump to 1.1.0

### PR DESCRIPTION
**Pull request by bios21**
Update to deno 1.1.0 and std 0.57.0

### CODE REVIEW

**Comment by bios21** [comment]
```
$ docker run test-runner         
Compile file:///src/tests/.deno.test.ts
running 25 tests
test test_badlock ... FAILED (2904ms)
test test_bundle ... ok (881ms)
test test_compile_error ... ok (910ms)
test test_config ... ok (1989ms)
test test_config2 ... ok (2008ms)
test test_console ... ok (2066ms)
test test_console_prefix ... ok (2075ms)
test test_console_prefix2 ... ok (2024ms)
test test_console_prefix3 ... ok (2026ms)
test test_console_prefix4 ... ok (2074ms)
test test_context ... ok (2034ms)
test test_denodir ... ok (1904ms)
test test_error ... ok (2209ms)
test test_example_zip ... ok (2263ms)
test test_importmap ... ok (2223ms)
test test_js ... ok (919ms)
test test_lock ... ok (1883ms)
test test_missing_handler ... ok (4380ms)
test test_noargs ... ok (2057ms)
test test_not_json ... ok (2007ms)
test test_run_deno ... ok (2068ms)
test test_self_contained ... ok (1343ms)
test test_simple ... ok (2057ms)
test test_wrongargs ... ok (2052ms)
test test_xray ... ok (2184ms)

failures:

test_badlock
AssertionError: Values are not equal:


    [Diff] Actual / Expected


    [
      {
        status: "error",
-       content: "{"errorMessage" : "lock file error: Subresource integrity check failed --lock=badlock.json
-   https://d..."
+       content: "{"errorMessage" : "lock file error: Compile file:///var/task/pad.ts
+   Subresource integrity check fail..."
      }
    ]

    at assertEquals (https://deno.land/std@0.57.0/testing/asserts.ts:170:9)
    at fn (file:///src/tests/test.ts:70:7)
    at async asyncOpSanitizer ($deno$/testing.ts:36:5)
    at async Object.resourceSanitizer [as fn] ($deno$/testing.ts:70:5)
    at async TestRunner.[Symbol.asyncIterator] ($deno$/testing.ts:275:11)
    at async Object.runTests ($deno$/testing.ts:358:20)

failures:

        test_badlock

test result: FAILED. 24 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out (50547ms)

```

**Comment by bios21** [comment]
So I fixed the badlock test, but I don't think this is the best way to do it since the test expects the the file to be already compiled.

File: tests/Dockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ RUN sh artifacts \
 
 WORKDIR /src/tests
 # Note: We have to run since fetch doesn't write!
-RUN deno run --lock-write --lock=lock.json pad.ts \
+RUN deno cache --lock-write --lock=lock.json pad.ts \
```
Comment (hayd) [inline tests/Dockerfile:43]: cool, this didn't used to work!

**Comment by hayd** [comment]
Thanks again @bios21 !

File: tests/Dockerfile (hunk 1 above)
Comment (bios21) [reply tests/Dockerfile:43]: Yeah, I don't really know when it was "fixed" but yeah now it's ok :)

**Comment by bios21** [comment]
No problem 👌

File: tests/Dockerfile (hunk 1 above)
Comment (hayd) [reply tests/Dockerfile:43]: I suppose should delete the comment above :)

**Pull request merged**